websites/O/OpenFING/presence.ts

```diff
@@ -0,0 +1,48 @@
+const presence = new Presence({
+	clientId: "968736771061985410",
+});
+
+let iFrameTime = -1;
+
+presence.on("iFrameData", (data: number) => {
+	iFrameTime = data;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "openfing",
```

Comment:
We use image URLs rather than Discord asset keys for maintainability
```suggestion
		largeImageKey: "https://i.imgur.com/7mO4xCK.png",
```